packages/webpack-common/src/extractText.js

```diff
@@ -18,7 +18,7 @@ export {
  */
 function getLoaderConfigByType (context, webpackConfig, fileType) {
   const loaderConfig = webpackConfig.module.loaders.find(
-    (loader) => loader.test === context.fileType(fileType)
+    (loader) => regexEqual(loader.test, context.fileType(fileType))
```

Comment:
I don't think we have to do a real regex comparison. `context.fileType(fileType)` should return the same regex reference throughout the whole configuration process.

The only scenario I can think of where it makes a difference: If there are two loader configs matching two file types which have semantically equal regexs to match. But I think we should not even match those as well 😉